build.gradle.kts

```diff
@@ -89,6 +91,12 @@ task<Jar>("testJar") {
     })
 }
 
+configure<PitestPluginExtension> {
+    testSourceSets = setOf(java.sourceSets["test"]?.apply {
+        java.exclude("rxbroadcast/integration/**")
```

Comment:
This is the issue with the build—this actually removes those files from the source set globally